go/interpreter/lfvm/interpreter.go

```diff
@@ -604,15 +603,12 @@ func steps(c *context, oneStepOnly bool) {
 	}
 }
 
-func satisfiesStackRequirements(c *context, op OpCode) bool {
+// satisfiesStackRequirements checks that the opCode will not make an out of
+// bounds access with the current stack size.
+// Caller should handle false return as an error.
```

Comment:
Why don't you directly return an error?

Reply:
because I wanted to keep changes to a minimum. Also with the function name starting with `satisfies` seems reasonable it returns true/false.